region/brinstar/pink/Big Pink.json

```diff
@@ -1955,6 +1959,28 @@
         "Refreeze the Zeb below the crumble blocks and ice clip up to reach the door."
       ]
     },
+    {
+      "link": [13, 5],
+      "name": "Big Pink Blind Zeb Ice Clip (Preserve Flash Suit)",
+      "notable": true,
+      "requires": [
+        "canStaggeredWalljump",
+        "canCameraManip",
+        "canOffScreenMovement",
+        "Wave",
+        "canInsaneJump",
+        "h_canHighPixelIceClip"
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Raise a Zeb to be just below the crumble blocks and blindly freeze it to set up an ice clip to reach the Power Bomb room exit door.",
+        "Get a bug from the pipe to spawn facing left and freeze it while moving to the ledge below the Hopper room door.",
+        "Go above, then blindly freeze it with a diagonal downward shot at a pixel perfect height.",
+        "Go back down and shoot it again to freeze it with a diagonal upward shot while on the ground to avoid losing the flash suit.",
+        "If the Zeb is frozen again, but not in a usable spot, it is possible to wall jump in the shaft below the crumble and refreeze it with a diagonal downward shot.",
```

Comment:
This part was confusing.  Is it at the correct height but inside of a wall or the middle of the shaft?